src/main/java/org/openmaptiles/layers/Transportation.java

```diff
@@ -362,13 +466,11 @@ public void process(Tables.OsmHighwayLinestring element, FeatureCollector featur
         .setAttr(Fields.CLASS, highwayClass)
         .setAttr(Fields.SUBCLASS, highwaySubclass(highwayClass, element.publicTransport(), highway))
         .setAttr(Fields.NETWORK, networkType != null ? networkType.name : null)
-        // TODO: including brunnel at low zooms leads to some large 300-400+kb z4-7 tiles, instead
-        //       we should only set brunnel if the line is above a certain length
-        .setAttr(Fields.BRUNNEL, brunnel(element.isBridge(), element.isTunnel(), element.isFord()))
+        .setAttrWithMinzoom(Fields.BRUNNEL, brunnelValue, brunnelMinzoom)
         // z8+
         .setAttrWithMinzoom(Fields.EXPRESSWAY, element.expressway() && !"motorway".equals(highway) ? 1 : null, 8)
         // z9+
-        .setAttrWithMinzoom(Fields.LAYER, nullIfLong(element.layer(), 0), 9)
+        .setAttrWithMinzoom(Fields.LAYER, nullIfLong(element.layer(), 0), brunnelValue == null ? 9 : brunnelMinzoom)
```

Comment:
I can't find where in the [OMT diff](https://github.com/openmaptiles/openmaptiles/compare/v3.14..master) we want to only show layer if brunnel is present? From what I can tell from the diff and https://github.com/openmaptiles/openmaptiles/pull/1283 we want brunnel to show up if >3px down to z4, and layer to show up if >3px down to z9

Reply:
cc/ @ZeLonewolf who made the change in openmaptiles